lib/internal/crypto/cipher.js

```diff
@@ -104,20 +108,12 @@ function createCipherWithIV(cipher, key, options, decipher, iv) {
   validateString(cipher, 'cipher');
   key = toBuf(key);
   if (!isArrayBufferView(key)) {
-    throw new ERR_INVALID_ARG_TYPE(
-      'key',
-      ['string', 'Buffer', 'TypedArray', 'DataView'],
-      key
-    );
+    throwInvalidArrayBufferView('key', key);
```

Comment:
Nit: I would prefer constructing the error object in the function and throwing it explicitly here. I think it would be better during the reviews and one less line in the stack trace 😉.

Reply:
That makes sense. I'll update shortly. Thanks!